Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN apt-get update && apt-get install -y trac python-babel \
 RUN pip install --upgrade Babel Trac
 RUN mkdir -p $TRAC_DIR
 RUN trac-admin $TRAC_DIR initenv $TRAC_PROJECT_NAME $DB_LINK
-RUN trac-admin $TRAC_DIR deploy $TRAC_DIR
+RUN trac-admin $TRAC_DIR deploy /tmp/deploy
```

Comment:
Actualy I would like to change to below,

```
RUN trac-admin /tmp/deploy initenv $TRAC_PROJECT_NAME $DB_LINK
RUN trac-admin /tmp/deploy deploy $TRAC_DIR
```

Reply:
Couldn't get your suggested changes to work without build errors :(
Could you please verify it works for you as expected and if yes provide the whole Dockerfile in case I'm missing something?